python_modules/dagster-graphql/dagster_graphql/schema/inputs.py

```diff
@@ -137,9 +137,36 @@ class Meta:
         name = "PartitionSetSelector"
 
 
+class GraphenePartitionRangeSelector(graphene.InputObjectType):
+    start = graphene.NonNull(graphene.String)
+    end = graphene.NonNull(graphene.String)
+
+    class Meta:
+        description = """This type represents a partition range selection with start and end."""
+        name = "PartitionRangeSelector"
+
+
+class GraphenePartitionsSelector(graphene.InputObjectType):
+    range = graphene.NonNull(GraphenePartitionRangeSelector)
+
+    class Meta:
+        description = """This type represents a partitions selection."""
+        name = "PartitionsSelector"
+
+
+class GraphenePartitionsByAssetSelector(graphene.InputObjectType):
```

Comment:
thoughts on renaming to `GraphenePartitionRangeByAsset`?

Or `GraphenePartitionRangeSelectionByAsset`?

Reply:
Keeping as is since we agreed on having nested `partitions` and `range`